static/redfish/v1/JsonSchemas/Fan/Fan.json

```diff
@@ -114,7 +114,6 @@
                 "Manufacturer": {
                     "description": "The manufacturer of this fan.",
                     "longDescription": "This property shall contain the name of the organization responsible for producing the fan.  This organization may be the entity from whom the fan is purchased, but this is not necessarily true.",
-                    "readonly": true,
```

Comment:
Why are we changing this..? From https://redfish.dmtf.org/schemas/Fan.v1_1_0.json

```
              "Manufacturer": {
                    "description": "The manufacturer of this fan.",
                    "longDescription": "This property shall contain the name of the organization responsible for producing the fan.  This organization may be the entity from whom the fan is purchased, but this is not necessarily true.",
                    "readonly": true,
                    "type": [
                        "string",
                        "null"
                    ]
                },
```

Reply:
while fixing the conflicts i think i have accidentally deleted it. I have restored it back